YeelightAPI/Device.cs

```diff
@@ -443,7 +443,8 @@ private async Task Watch()
                                                     ICommandResultHandler commandResultHandler;
                                                     lock (_currentCommandResults)
                                                     {
-                                                        commandResultHandler = _currentCommandResults[commandResult.Id];
+                                                        if (!_currentCommandResults.TryGetValue(commandResult.Id, out commandResultHandler))
+                                                            continue; // ignore if the result can't be found
```

Comment:
Won't happen when using unique ids, but this could happen if someone does multiple calls providing the same `id` via `device.ExecuteCommandWithResponse(...)` or `device.ExecuteCommand(...)` (this wouldn't happen if `id` was internal to the library and `ExecuteCommand` was deprecated)